shell.nix

```
@@ -0,0 +1,22 @@
+{ sources ? import ./nix/sources.nix }:
+let
+  nixpkgs = import sources.nixpkgs { };
+in
+nixpkgs.mkShell {
+  name = "bellroy-gem-env";
+  buildInputs = with nixpkgs; [
+    bundler
+    libnotify
+    niv
+    nodejs-10_x
+    pkg-config
+    postgresql_11
+    readline
+    ruby_2_7
+    zlib
+  ]
+  ++ (nixpkgs.lib.optionals stdenv.hostPlatform.isDarwin [ libiconv darwin.apple_sdk.frameworks.CoreServices ]);
```

You're actually in the scope of the `with nixpkgs`, so you could even simplify to:
```suggestion
  ++ (lib.optionals stdenv.hostPlatform.isDarwin [ libiconv darwin.apple_sdk.frameworks.CoreServices ]);
```